feature/office/src/main/java/com/iteneum/office/presentation/ui/OfficeViewModel.kt

```diff
@@ -0,0 +1,93 @@
+package com.iteneum.office.presentation.ui
+
+import androidx.compose.runtime.mutableStateListOf
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.iteneum.OfficeInfoItemModel
+import com.iteneum.network.DataState
+import com.itenum.domain.ContactTelephone
+import com.itenum.domain.Email
+import com.itenum.domain.IintentActions
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+/**
+ * This class will handle the UI Logic and expose data to the composable views.
+ * @param intentActions needs to be injected in order to decouple logic *
+ * TODO implement @param viewModel the view model we are using inside our view compose has to be injected
+ * /*viewModel: OfficeViewModel = hiltViewModel()*/
+ * @author Andres Ivan Medina
+ */
+@HiltViewModel
+class OfficeViewModel @Inject constructor(val intentActions: IintentActions) : ViewModel() {
```

Comment:
OfficeViewModel duplicate

<img width="334" alt="image" src="https://github.com/irv205/droid_leaspert/assets/42255248/a4272a28-09ab-42fe-9335-2e8ada2572a8">

Reply:
will remove it,